### DISCUSSION TITLE
Reading and selection from multiple data input files, one policy rego file

### DISCUSSION THREAD

**Discussion by jwanggg** [open-policy-agent/community#273] [OPA and Rego] [answered]
For OPA, is it possible to selection one file from multiple data input jsons?

In the rules is it possible to specifically select in input to read from?

For example I have added groups1.json and groups2.json to the bundle. (both json are same format)

In the rego policy file can I use the input to select a specific input json file to read from?

something like
```
data := input.file
```
so the rest of the rules only use that file

**Comment by anderseknert** [comment]
Maybe something like [dynamic policy composition](https://www.styra.com/blog/dynamic-policy-composition-for-opa/) would be what you're looking for? If not, could you elaborate some on what you're trying to do, and perhaps include some policy and data?

**Comment by jwanggg** [comment, marked as answer]
I was able to split the data into different directories and put them into bundles which the policy was able to read correctly
